src/isIsogram.test.js

```diff
@@ -1,9 +1,29 @@
-'use strict';
-
+// eslint-disable-next-line strict
 describe('isIsogram', () => {
   const { isIsogram } = require('./isIsogram');
 
-  it(`should be declared`, () => {
+  it('should be a function', () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
+
+  it('should return true for empty string', () => {
+    expect(isIsogram('')).toBe(true);
+  });
+
+  it('should return true for isograms', () => {
+    expect(isIsogram('isogram')).toBe(true);
+    expect(isIsogram('Unique')).toBe(false);
+    expect(isIsogram('Alphabetic')).toBe(false);
+  });
+
+  it('should return false for non-isograms', () => {
+    expect(isIsogram('hello')).toBe(false);
+    expect(isIsogram('Dermatoglyphics')).toBe(true);
```

Comment:
```suggestion
    expect(isIsogram('Dermatoglyphics')).toBe(true);
```
move this line to the test above